src/comp/ScrollableTagList.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+
+import style from "css/comp/ScrollableTagList.module.css";
+import ProjectTag from "./ProjectTag";
+
+type ProjectTagListProps = {
+  tagIDs: string[];
+  onTagClick?: (
+    tagID: string,
+    e: React.MouseEvent<HTMLParagraphElement>
+  ) => void;
+};
+
+export class ScrollableTagList extends React.Component<
+  ProjectTagListProps,
+  {}
+> {
+  render() {
+    return (
+      <div className={style.tag_list_wrapper}>
+        <div className={style.tag_list}>
```

Comment:
```suggestion
      <>
```
他の項目と動作が同じになるようにした方がいいです